spacetime/tests/test_linear_scalar.py

```diff
@@ -127,10 +130,111 @@ def _march():
 
         svr2 = self._build_solver(self.resolution)[-1]
 
-        for it in range(self.nstep*self.cycle):
+        for it in range(self.nstep * self.cycle):
             _march()
             svr2.march_alpha2(steps=1)
-            self.assertEqual(self.svr.get_so0(0).ndarray.tolist(),
-                             svr2.get_so0(0).ndarray.tolist())
+            self.assertEqual(
+                self.svr.get_so0(0).ndarray.tolist(), svr2.get_so0(0).ndarray.tolist()
+            )
+
+
+class NormTC(unittest.TestCase):
```

Comment:
Since this is to test for the error of the linear solver, I suggest to rename this to LinearScalarErrorTC.

Norm is a mathematical term and not specific to error analysis.  `NormTC` the name is ambiguous and misleading.